paypal/express/views.py

```diff
@@ -374,18 +402,32 @@ def get_shipping_method(self, basket, shipping_address=None, **kwargs):
         if not basket.is_shipping_required():
             return NoShippingRequired()
 
-        # Instantiate a new FixedPrice shipping method instance
+        code = self.checkout_session.shipping_method_code(basket) # why is code none for valid orders and showing the not shipping
+        shipping_method = Repository().get_default_shipping_method(
+            user=self.request.user, basket=basket,
+            shipping_addr=shipping_address, request=self.request)
+
+
+        if shipping_address.country.pk not in [country.pk for country in \
+                                        shipping_method.countries.all()]:
+            countries = ", ".join([country.pk for country in \
```

Comment:
Can we assign a variable for shipping_method.countires.all() so we don't have to make the query twice :)